src/components/Tabs/Tab/Tab.tsx

```diff
@@ -47,10 +53,16 @@ export const Tab = React.forwardRef<HTMLLIElement, TabProps>((props, ref) => {
       id={item.id}
       onClick={onSelectTab}
       onKeyDown={onKeyDown}
```

Comment:
In order to fix selecting the anchor using the `space` key, we need to also adjust the `onKeyDown` callback.
We just need to **not** prevent default if tab has an anchor.